modules/microshift-otel-config.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+//  microshift_running_apps/microshift-observability-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-otel-config_{context}"]
+= Configuring {microshift-short} Observability
+
+You can configure {microshift-short} Observability after it is installed to identify a valid endpoint.
+
+[IMPORTANT]
+====
+You must specify an OpenTelemetry Protocol (OTLP)-compatible endpoint for each configuration, such as Prometheus, before starting {microshift-short}. If an endpoint is not specified, {microshift-short} Observability will not start.
+====
+
+.Procedure
+
+. Update the `/etc/microshift/observability/opentelemetry-collector.yaml` file to specify an OTLP-compatible endpoint with the following information. The endpoint must link to an IP address or host name, and port number of an OTLP service.
```

Comment:
You don't need to use OTLP exporter, it's not mandatory. You can use other exporters, see this: https://opentelemetry.io/docs/collector/configuration/#exporters